.github/workflows/go.yml

```diff
@@ -76,7 +76,12 @@ jobs:
         cd ..
 
     - name: COA Test
-      run: cd coa && mage cleanTest
+      run: |
```

Comment:
Do need to do like this, you can just mv go.work.bk to go.work. https://github.com/eclipse-symphony/symphony/blob/034b25fc13f9de80a6345dfcd724d683eb69876a/.github/workflows/integration.yml#L83C1-L85C30

More importantly, I double you have some missing dependencies in go.mod in COA. When running unit test, usually it is not needed to add go.work.

Reply:
I have updated go.mod in COA